chat-plugins/info.js

```diff
@@ -1357,6 +1357,7 @@ exports.commands = {
 			"- /tour create <em>format</em>, roundrobin: Creates a new round robin tournament in the current room.<br />" +
 			"- /tour end: Forcibly ends the tournament in the current room<br />" +
 			"- /tour start: Starts the tournament in the current room<br />" +
+			"- /tour banlist [pokemon], [talent], [...]: After the tournament create, banlist move, ability, pokemon, item." +
```

Comment:
I'd personally do `After the tournament's creation, bans moves, abilities, Pokémon or items from being used in it.`